EventListener/ClickjackingListener.php

```diff
@@ -30,8 +31,16 @@ public static function getSubscribedEvents()
         return array(KernelEvents::RESPONSE => 'onKernelResponse');
     }
 
-    public function onKernelResponse(FilterResponseEvent $e)
+    /**
+     * @param FilterResponseEvent|ResponseEvent $e
+     */
+    public function onKernelResponse($e)
     {
+        // Compatibility with Symfony < 5 and Symfony >=5
+        if (!$e instanceof FilterResponseEvent && !$e instanceof ResponseEvent) {
+            return;
```

Comment:
this should be a `throw new TypeError` (or `throw new InvalidArgumentException` if PHP 5.x support is needed) rather than a silent failure. Otherwise, switching back to a typehint when dropping support for Symfony `< 4.4` would be a BC break.

Btw, I suggest tagging all listener classes as `@final`, otherwise this typehint removal is a BC break.